java-generator/core/src/main/java/io/fabric8/java/generator/nodes/AbstractJSONSchema2Pojo.java

```diff
@@ -35,10 +36,24 @@
     static final String OBJECT_CRD_TYPE = "object";
     static final String ARRAY_CRD_TYPE = "array";
 
+    private final Optional<String> description;
```

Comment:
Despite this PR guards the field initialization, it is highly unadvised to use Optionals as field types and method parameters.

IMHO it would be better to declare the field as `private final String description;` and then consume it in `JObject#generateJava` (L252-L253) with `if (Utils.isNotBlankOrNull(prop.getDescription())) {`

Reply:
Thanks for your input @manusa - will fix!

Reply:
And... fixed :)